### ISSUE TITLE
How specific to Spark is this project?

### ISSUE THREAD

**Issue by SyniaN** [sparknz/Base-Tokens#2]
Considering this is an open source project. Are we expecting other people ( not spark employee or contractor) to use this in their own projects?

**Comment by miketamis** [comment]
This repo I have made the base repo, the tokens in this repo are the ones from @NoWorries white label brand, so this repo can be fork by any organisation to kick off their design tokens, I have forked this repo into SET-Tokens https://github.com/sparknz/SET-Tokens which is spark specific and really won't be used by other organisations directly.

Why is that open source (which is more your original question as it was made when this repo was still the SET-Token)? because it means we are open about our way of working, to make it easy for anyone around the business to contribute and feel comfortable contribute, to show off to potential future employees of spark that we are forward thinking in our development, to be able share our ideas and be involved in the global discussion on what is the best way to build a design system. Heres a bunch of other companies that do it: https://github.com/alexpate/awesome-design-systems
